solutions/logging/core-experimentation/cloud-logging-buckets.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+######
+# Cloud Logging bucket for Security logs: Cloud Audit, Access Transparency Logs, and Data Access Logs
+# Logs are routed using a log sink to a central logging project into a dedicated log bucket
+# AU-4(1), AU-9(2) - Log buckets are created in a logging project, isolating them from the source of the log entries in other projects
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
+kind: LoggingLogBucket
+metadata:
+  name: security-log-bucket
+  namespace: logging
+  annotations:
+    config.kubernetes.io/depends-on: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/${logging-prj-id} # kpt-set: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/${logging-prj-id}
+spec:
+  projectRef:
+    name: projects/logging-prj-id # kpt-set: ${logging-prj-id}
```

Comment:
to avoid possible reader confusion (for the expected projectRef.name format)...should we rename:
`name: projects/logging-prj-id # kpt-set: ${logging-prj-id}`
to
`name: logging-prj-id # kpt-set: ${logging-prj-id}`
?

Reply:
Fixed, please resolve the comment if satisfied.

Reply:
sorry, forgot to mention that it was also in a few other files...still some remaining in: 
- client-experimentation/project-iam.yaml
- core-experimentation/cloud-logging-buckets.yaml
- core-experimentation/project-iam.yaml

Reply:
Good catch. Thanks. Fixed, please resolve the comment if satisfied.